client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -298,6 +298,13 @@ export default function getThankYouPageUrl( {
 		return `/checkout/akismet/thank-you/${ productSlug }`;
 	}
 
+	// A4A client checkout uses a custom thank you page
```

Comment:
The output of this function is completely covered by tests (see the top of the file) so if you could add a test(s) for this behavior, it would be appreciated and will help to prevent regressions.